backend/app/api/v1/lantern_api.py

```
@@ -1,54 +1,49 @@
+import re
```

Flake8 (7.2.0)

[error] 're' imported but unused

(F401)